### DISCUSSION TITLE
Maui 8 preview 6  can anyone create a blank project using windows host

### DISCUSSION THREAD

**Discussion by developer9969** [dotnet/maui#16310] [General]
Hi I have tried since the last 2 previews to move to .net 8 mainly because as we all know many fixes will never make it to .net 7
However even creating a blank app is full of issues , previous preview android was fine but iOS did not launch , now both dont build ..

has anyone got it working , I mean creating a blank app ????

thanks

**Comment by JoacimWall** [comment]
Hi I was able to get empty iOS to work on Mac and simulator. But as sone as us try to build a more complex solution you get error that are looking like tooling issues. 

this has bin the case on all preview just new error messages on new releases.  

//Joacim

**Reply by developer9969** [reply]
Hi have u tried using windows ? I am getting all sorts of issues
